koma-core-api/common/src/koma/internal/default/generated/ndarray/DefaultLongNDArray.kt

```diff
@@ -97,6 +97,53 @@ open class DefaultLongNDArray(@KomaJsName("shape_private") vararg protected val
     }
 
 
+    override fun toString(): String {
+        if (size == 0)
+            return "[]"
+        val s = StringBuilder()
+        val index = IntArray(shape.size)
+        val lastAxis = shape.size-1
+        val width = shape[lastAxis]
+        var printBracketsFrom = 0
+        while (index[0] < shape[0]) {
+            for (i in 0..lastAxis)
+                s.append(if (i >= printBracketsFrom) '[' else ' ')
```

Comment:
One goal of the current `.toString` implemented for matrices is to generate a string that is copy+pasteable into code. For example, using the EJML backend `println(zeros(3,3))` yields:

```
mat[ 0.00,  0.00,  0.00 end
     0.00,  0.00,  0.00 end
     0.00,  0.00,  0.00 ]
```

This has some utility in debugging, as one can copy and paste an output they have into some code and run analysis against it. I'm wondering what you'd think of converting this `NDArray.toString` method into something similar, i.e. letting

```
println(ndArrayOf(1, 2, 3, shape = intArrayOf(2,2)))
```
produce `ndArrayOf(1, 2, 3, shape = intArrayOf(2,2))` as its output? I realize its less clean and I'm not totally against the cleaner output you have here, but curious what everyone prefers.

Reply:
Sounds good to me. I think I originally went down that road with a `repr` method before converging to everything being `toString`. There are still vestiges of the [old repr method in code](https://github.com/kyonifer/koma/blob/master/koma-core-api/common/src/koma/matrix/Matrix.kt#L255), so perhaps it can be revitalized on the NDArray type (and removed from matrix).

Reply:
Opened #79 to track this issue separately from this PR.